src/components/Filters/index.js

```diff
@@ -0,0 +1,33 @@
+import FilterBtn from "./filter-btn";
+import './index.css';
+
+function Filters(props) {
+    const filters = [
+        {text: 'All'},
+        {text: 'Active'},
+        {text: 'Completed'}
+    ];
+
+    const handleRadioChange = (e) => {
+        const value = e.target.value === 'Active' ? false : e.target.value === 'Completed' ? true : e.target.value;
```

Comment:
Try to avoid this kind of multiple ternary operators. Better to rewrite it with if else.